### PR TITLE
fixed test 'test_qsub_with_option_a' from 'TestQsubOptionsArguments' failed due to race condition.

### PR DESCRIPTION
**Pull request by dinesh1306**
#### Describe Bug or Feature
currently execution time is formatted till minute i.e. DD:MM:YYYY:HH:MM.
Race condition such that in test we calculate future execution time by adding 60 sec. in current time, but when current time is near to end for minute consider current minute value due to job immediately comes in R state, but PTL expect job should be in W state due test got failed.

#### Describe Your Change
increase future time calculation value. change value from 60 to 120sec. due to at least we get 1 min time period and we can check expected job state as W. 

#### Attach Test and Valgrind Logs/Output
Before Fix:
[PTL_log.txt](https://github.com/openpbs/openpbs/files/5830759/PTL_log.txt)

After Fix:
[res_after_fix_1.txt](https://github.com/openpbs/openpbs/files/5830764/res_after_fix_1.txt)
[res_after_fix_2.txt](https://github.com/openpbs/openpbs/files/5830778/res_after_fix_2.txt)
[res_after_fix_3.txt](https://github.com/openpbs/openpbs/files/5830777/res_after_fix_3.txt)
[res_after_fix_4.txt](https://github.com/openpbs/openpbs/files/5830772/res_after_fix_4.txt)
[res_after_fix_5.txt](https://github.com/openpbs/openpbs/files/5830774/res_after_fix_5.txt)
[res_after_fix_6.txt](https://github.com/openpbs/openpbs/files/5830775/res_after_fix_6.txt)
[res_after_fix_7.txt](https://github.com/openpbs/openpbs/files/5830784/res_after_fix_7.txt)
[res_after_fix_8.txt](https://github.com/openpbs/openpbs/files/5830783/res_after_fix_8.txt)
[res_after_fix_9.txt](https://github.com/openpbs/openpbs/files/5830780/res_after_fix_9.txt)
[res_after_fix_10.txt](https://github.com/openpbs/openpbs/files/5830787/res_after_fix_10.txt)

### CODE REVIEW

**Review by shilpakodli — APPROVED** [review]
Changes LGTM.

File: test/tests/functional/pbs_qsub_opts_args.py [hunk 1]
```diff
@@ -170,16 +170,16 @@ def test_qsub_with_option_a(self):
...
         try:
             jid_1 = self.server.submit(j1)
         except PbsSubmitError as e:
             self.validate_error(e)
         self.server.expect(JOB, {'job_state': 'W'}, id=jid_1)
         self.logger.info(
-            'waiting for 60 seconds to run the job as it is a future job...')
-        self.server.expect(JOB, {'job_state': 'R'}, id=jid_1, offset=60)
+            'waiting for 90 seconds to run the job as it is a future job...')
+        self.server.expect(JOB, {'job_state': 'R'}, id=jid_1, offset=90)
```
Comment (bhroam) [inline test/tests/functional/pbs_qsub_opts_args.py:182]: Since the job might have to wait up to 30s to run, consider adding an interval of 1 or 2s.

**Review by bhroam — APPROVED** [review]
I'll sign off.  I have one minor comment you can consider doing or not.

**Review by shilpakodli — APPROVED** [review]

**Comment by dinesh1306** [comment]
@bhroam , I have incorporate review comment. Please have a look.
[res_after_incorporate_comments_1.txt](https://github.com/openpbs/openpbs/files/5847499/res_after_incorporate_comments_1.txt)
[res_after_incorporate_comments_2.txt](https://github.com/openpbs/openpbs/files/5847500/res_after_incorporate_comments_2.txt)
[res_after_incorporate_comments_3.txt](https://github.com/openpbs/openpbs/files/5847501/res_after_incorporate_comments_3.txt)
[res_after_incorporate_comments_4.txt](https://github.com/openpbs/openpbs/files/5847502/res_after_incorporate_comments_4.txt)
[res_after_incorporate_comments_5.txt](https://github.com/openpbs/openpbs/files/5847498/res_after_incorporate_comments_5.txt)

**Pull request merged**